### PR TITLE
Add CodingBlocks

### LINKED ISSUE #411: Hacktoberfest 2019: CodingBlocks
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : CodingBlocks

Q. Where have you seen the announcement (URL)?  
A. : https://blog.codingblocks.com/2019/cb-hacktoberfest-2019/

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, bag

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by driver-by**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Adding Coding Blocks 
Fixes #411

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T18:44:53.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "CodingBlocks",
+    "difficulty": "medium",
+    "description": "Have one merged PR on any of <a href=\"https://cb.lk/gh\">Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag (1 PR - t-shirt, 5+ PRs - bag)",
+    "reference": "https://blog.codingblocks.com/2019/cb-hacktoberfest-2019/",
+    "image": "https://blog.codingblocks.com/content/images/size/w2000/2019/10/asc.jpg",
```
Comment (aslafy-z) [inline data.json]: I suggest using only the left side of the image. Something like this: https://i.imgur.com/0SrSsNM.png

Comment (driver-by) [reply data.json]: Do you wan to use imgur as source for image? Looks like not reliable source of information for me :) However every website could remove image used in data.json, so maybe imgur is even more reliable :)

Comment (aslafy-z) [reply data.json]: From my experience, imgur is one of the most robust host. Anyway, we download the image and serve it ourselves.

Comment (driver-by) [reply data.json]: Got it. By "not reliable" I meant you can always clean up your imgur images and accidentally remove this nice image :)
Changed the image with following commit btw.

Comment (aslafy-z) [reply data.json]: I always create them in non-logged mode, so I'm sure this won't happen :D

Comment (aslafy-z) [reply data.json]: It looks nice, thanks!

File: data.json [hunk 2]
```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T18:44:53.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "CodingBlocks",
+    "difficulty": "medium",
+    "description": "Have one merged PR on any of <a href=\"https://cb.lk/gh\">Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag (1 PR - t-shirt, 5+ PRs - bag)",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Have one merged PR on any of <a href='https://cb.lk/gh'>Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag (1 PR - t-shirt, 5+ PRs - bag)",
```

Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Have one merged PR on any of <a href='https://cb.lk/gh'>Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag for 5+",
```

Not sure about that one, wdyt?

File: data.json [hunk 3]
```diff
@@ -80,6 +80,15 @@
...
+    "reference": "https://blog.codingblocks.com/2019/cb-hacktoberfest-2019/",
+    "image": "https://i.imgur.com/0SrSsNM.png",
+    "dateAdded": "2019-10-09T19:00:00.000Z",
+    "tags": ["stickers", "hacktoberfest", "clothing"]
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "tags": ["accessories", "stickers", "hacktoberfest", "clothing"]
```

**Comment by aslafy-z** [comment]
Merging this without confirmation as it's already the end :( Thank you @driver-by

**Pull request merged**